nav2_dwb_controller/dwb_core/include/dwb_core/dwb_local_planner.hpp

```diff
@@ -201,6 +201,7 @@ class DWBLocalPlanner : public nav2_core::Controller
   bool debug_trajectory_details_;
   rclcpp::Duration transform_tolerance_{0, 0};
   bool shorten_transformed_plan_;
+  double shorten_distance_;
```

Comment:
New parameters must be added to our docs in the configuration page with a description and added to the example XML at the bottom (https://github.com/ros-planning/navigation.ros.org). Additionally, need a migration guide entry about the change for the new distributions

Reply:
Done in https://github.com/ros-planning/navigation.ros.org/pull/389